File: YouFly.web/ClientPortal/src/app/Transaction.ts
export class Transaction{
  constructor(
    public id: number,
    public ccNumber: number,
    public ccExp: number,    
    public userid: number,
    public flightid: number,
    public numOfBusTks: number,
    public numOfFCTks: number,
    public total: number
  ){}
}

File: YouFly.core/Models/Transaction.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace YouFly.core.Models
{
    public class Transaction
    {
        public int TransactionId { get; set; }
        public int CCNumber { get; set; }
        public int CCExp { get; set; }
        public int TotalPrice { get; set; }
        public int ConfimationNum { get; set; }

        public int UserId { get; set; }
        public User User { get; set; }

        public List<Ticket> Tickets { get; set; }
    }
}
